app/core/CocoVueRouter.js

```diff
@@ -0,0 +1,32 @@
+import VueRouter from 'vue-router'
+
+import SchoolAdminDashboard from 'app/views/school-administrator/SchoolAdministratorComponent'
+import SchoolAdminDashboardTeacherListView from 'app/views/school-administrator/teachers/SchoolAdminTeacherListView'
+import SchoolAdminTeacherView from 'app/views/school-administrator/dashboard/SchoolAdminDashboardTeacherView'
+
+import TeacherClassView from 'views/teachers/class/TeacherClassView'
```

Comment:
I was assuming this is the same backbone view until now when I realized this is pointing to the new `TeacherClassView.vue` which then renders the existing backbone view `TeachClassView.coffee`. 
I think it would be less confusing if we keep the names different (eg: `TeacherClassViewComponent.vue` or `TeacherClassVueView.vue` for the vue page)

Reply:
Also, why are we keeping in inside `/class` folder when the existing `TeachClassView.coffee` is in the `/courses` folder.

Reply:
Long-term, we'd probably want to end up with a `TeacherClassView` and have the default assumption be Vue.  for these imports, would it be enough to include the file extension to indicate which?

Reply:
Re file location - moved to courses

Re naming I think there are two parts to this discussion:

1. `.vue` extension - I think this is a great idea and an easy win in terms of adding additional information in places where we could possibly be confused by the import
1. Component naming - I think we should define a taxonomy for some of the different types of components we have.  For example we could break down into "top level pages", "smart components" and "dumb components" (roughly stealing from React / Redux).  It would be helpful to have a convention that gives us clues as to what the file might be.  Picking this up in slack for a quick discussion.